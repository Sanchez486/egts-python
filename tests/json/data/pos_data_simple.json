{
  "transport": {
    "pt": 1
  },
  "service": {
    "sdr": [
      {
        "rfl": {
          "ssod": 0,
          "rsod": 0
        },
        "tm": "2018-06-05T10:00:00",
        "sst": 2,
        "rst": 2,
        "rd": [
          {
            "srt": 16,
            "srd": {
              "ntm": "2018-06-05T10:00:01",
              "lat": -25.5246,
              "long": 153.65379,
              "lohs": 0,
              "lahs": 0,
              "mv": 1,
              "bb": 0,
              "cs": 0,
              "vld": 1,
              "fix": 0,
              "spd": 600,
              "alt": "0x010203",
              "alts": 0,
              "src": 13,
              "srcd": -100,
              "dir": 300,
              "odm": "0x050607",
              "din": "0b00000101"
            }
          }
        ]
      }
    ]
  }
}
