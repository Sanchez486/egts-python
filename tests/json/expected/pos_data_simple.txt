EGTS protocol validator v.1.0  Copyright JSC "Navigation-information systems", 2011
------------------------------------------------------------------------------------
File: 'D:\PycharmProjects\egts-python\tests\json\generated\tmp_msg.bin'
Packet data: 
 0100000B002800010001A51D000100042024D90F0202101A002124D90F4D729A483DA387DAD158822C050607050D0102039CFF9EBB


EGTS Transport Layer:
---------------------
  Validating result   - 0 (OK)

  Protocol Version    - 1
  Security Key ID     - 0
  Flags               - 00000000b (0x00)
       Prefix         - 00
       Route          -   0
       Encryption Alg -    00
       Compression    -      0
       Priority       -       00 (the highest)
  Header Length       - 11
  Header Encoding     - 0
  Frame Data Length   - 40
  Packet ID           - 1
  No route info       -
  Header Check Sum    - 0xA5

EGTS Service Layer:
---------------------
  Validating result   - 0 (OK)

  Packet Type         - EGTS_PT_APPDATA
  Service Layer CS    - 0xBB9E

    Service Layer Record:
    ---------------------
    Validating Result    - 0 (OK)

    Record Length               - 29
    Record Number               - 1
    Record flags                -     00000100b (0x04)
        Sourse Service On Device    - 0
        Recipient Service On Device -  0
        Group Flag                  -   0
        Record Processing Priority  -    00 (the highest)
        Time Field Exists           -      1
        Event ID Field Exists       -       0
        Object ID Field Exists      -        0
    Time                        - 0x0FD92420 (05.06.2018 10:00:00 GMT)
    Source Service Type         - 2 (EGTS_TELEDATA_SERVICE) 
    Recipient Service Type      - 2 (EGTS_TELEDATA_SERVICE) 

       Subrecord Data:
       ------------------
       Validating Result   - 150 (Unknown service)

       Subrecord Type      - 16 (unspecified)
       Subrecord Length    - 26


-------------------------------------------------------------
Press any key to exit...
